databaseQueries/programs.js

```diff
@@ -0,0 +1,179 @@
+/**
+ * name : programs.js
+ * author : Praveen
+ * created-date : 13-June-2024
+ * Description : program helper for DB interactions.
+ */
+
+// Dependencies 
+
+/**
+    * Programs
+    * @class
+*/
+
+
+
+module.exports= class Programs{
+   /**
+   * Programs Document.
+   * @method
+   * @name programDocuments
+   * @param {Array} [filterQuery = "all"] - solution ids.
+   * @param {Array} [fieldsArray = "all"] - projected fields.
+   * @param {Array} [skipFields = "none"] - field not to include.
+   * @param {Number} pageNo - page no.
+   * @param {Number} pageSize - page size.
+   * @returns {Array} List of programs.
+   */
+
+  static programDocuments(filterQuery = 'all', fieldsArray = 'all', skipFields = 'none', pageNo = '', pageSize = '') {
+    return new Promise(async (resolve, reject) => {
```

Comment:
@praveenKDass add comments for code readability

Reply:
Added @VISHNUDAS-tunerlabs